### ISSUE TITLE
Unstable Training

### ISSUE THREAD

**Issue by gumowangfei** [dmlc/dgl#1469]
def myinitializer(shape, dtype, ctx, id_range):
        return torch.zeros(shape, dtype=dtype, device=ctx)

I added g.set_n_initializer (self.myinitializer) 、g.set_e_initializer (self.myinitializer) and dgl.random.seed (random_state).

Use g.update_all(fn.copy_u('ft' ,'m'), fn.sum('m', 'h')) for each retraining. The results are consistent. 

Each retraining using g.update_all(fn.u_add_v('ft', 'ft', 'm'), fn.sum('m', 'h')) for each retraining. The results are inconsistent, and there seems to be random in the calculation ingredient.

I want to make the result of each training consistent, how do I do it?

**Comment by yzh119** [comment]
Hi, I think this issue is duplicate to #1471 and you can refer to my answer there.
